src/fsevent.rs

```diff
@@ -281,6 +281,10 @@ impl FsEventWatcher {
         unsafe {
             let mut err: cf::CFErrorRef = ptr::null_mut();
```

Comment:
we should probably investigate whether this should be handled instead of ignored (and deallocated)